targets/HUDL/main.cpp

```diff
@@ -13,53 +20,180 @@
 #include <HUDL/HUDL.hpp>
 
 namespace IO = EVT::core::IO;
+namespace DEV = EVT::core::DEV;
 namespace time = EVT::core::time;
+namespace log = EVT::core::log;
 using namespace std;
 
-const uint32_t SPI_SPEED = SPI_SPEED_4MHZ;// 4MHz
+const uint32_t SPI_SPEED = SPI_SPEED_500KHZ;
 const uint8_t deviceCount = 1;
 
+///////////////////////////////////////////////////////////////////////////////
+// EVT-core CAN callback and CAN setup. This will include logic to set
+// aside CANopen messages into a specific queue
+///////////////////////////////////////////////////////////////////////////////
+
+/**
+ * Interrupt handler to get CAN messages. A function pointer to this function
+ * will be passed to the EVT-core CAN interface which will in turn call this
+ * function each time a new CAN message comes in.
+ *
+ * NOTE: For this sample, every non-extended (so 11 bit CAN IDs) will be
+ * assumed to be intended to be passed as a CANopen message.
+ *
+ * @param message[in] The passed in CAN message that was read.
+ */
+
+IO::UART& uart = IO::getUART<IO::Pin::PB_10, IO::Pin::PB_11>(9600);
```

Comment:
I know you probably put this in to make testing easier and quicker, but I would much prefer we didn't have top-level variables in the main target for a board. Can you convert this to work with the logger instead?